Move relayer to subfolder (#411)
ava-labs/awm-relayer: README.md

# Avalanche ICM Off-chain Services

> This repository contains services related to Avalanche ICM (Interchain Messaging) systems along with shared code to support them. 

Reviewer: ```suggestion
This repository contains off-chain services that help support Avalanche Interchain Messaging (ICM).
```
Super nit on wording

Currently implemented applications are 

> 1. [AWM Relayer](relayer/README.md)

Reviewer: Should this be ICM relayer now?

May also be helpful to include a 1 sentence description of each service at the top level here.

Author: I'll include descriptions but do we want to hold off on changing the name here until we change the name of the build binary / rest of the documentation / and the whole repo?

Reviewer: Ah yeah good call
